Rebuilding when installing from npm is broken on 0.9.0

Caused by the fix of #7 

```
Run npm install --force node-magickwand@0.9.0 --build-from-source
npm WARN using --force Recommended protections disabled.
npm ERR! code 1
npm ERR! path /home/runner/work/node-magickwand/node-magickwand/node_modules/node-magickwand
npm ERR! command failed
npm ERR! command sh -c node-pre-gyp install --fallback-to-build && npx copyfiles -sf swig/*.d.ts lib
npm ERR! Failed to execute '/opt/hostedtoolcache/node/18.17.1/x6[4](https://github.com/mmomtchev/node-magickwand/actions/runs/6186789692/job/16795209879#step:7:5)/bin/node /opt/hostedtoolcache/node/18.17.1/x64/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/home/runner/work/node-magickwand/node-magickwand/node_modules/node-magickwand/lib/binding/linux-x64/node-magickwand.node --module_name=node-magickwand --module_path=/home/runner/work/node-magickwand/node-magickwand/node_modules/node-magickwand/lib/binding/linux-x64 --napi_version=9 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v108' (1)
npm ERR! node-pre-gyp info it worked if it ends with ok
npm ERR! node-pre-gyp info using node-pre-gyp@1.0.11
npm ERR! node-pre-gyp info using node@18.17.1 | linux | x64
npm ERR! node-pre-gyp info build requesting source compile
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.3.1
npm ERR! gyp info using node@18.17.1 | linux | x64
npm ERR! gyp info ok 
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.3.1
npm ERR! gyp info using node@18.17.1 | linux | x64
npm ERR! gyp info find Python using Python version 3.8.10 found at "/usr/bin/python3"
npm ERR! gyp http GET https://nodejs.org/download/release/v18.17.1/node-v18.17.1-headers.tar.gz
npm ERR! gyp http 200 https://nodejs.org/download/release/v18.17.1/node-v18.17.1-headers.tar.gz
npm ERR! gyp http GET https://nodejs.org/download/release/v18.17.1/SHASUMS2[5](https://github.com/mmomtchev/node-magickwand/actions/runs/6186789692/job/16795209879#step:7:6)[6](https://github.com/mmomtchev/node-magickwand/actions/runs/6186789692/job/16795209879#step:7:7).txt
npm ERR! gyp http 200 https://nodejs.org/download/release/v18.1[7](https://github.com/mmomtchev/node-magickwand/actions/runs/6186789692/job/16795209879#step:7:8).1/SHASUMS256.txt
npm ERR! gyp info spawn /usr/bin/python3
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/opt/hostedtoolcache/node/1[8](https://github.com/mmomtchev/node-magickwand/actions/runs/6186789692/job/16795209879#step:7:9).[17](https://github.com/mmomtchev/node-magickwand/actions/runs/6186789692/job/16795209879#step:7:18).1/x64/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/runner/work/node-magickwand/node-magickwand/node_modules/node-magickwand/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/opt/hostedtoolcache/node/[18](https://github.com/mmomtchev/node-magickwand/actions/runs/6186789692/job/16795209879#step:7:19).17.1/x64/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/runner/.cache/node-gyp/18.17.1/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/home/runner/.cache/node-gyp/18.17.1',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/opt/hostedtoolcache/node/18.17.1/x64/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/home/runner/.cache/node-gyp/18.17.1/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/home/runner/work/node-magickwand/node-magickwand/node_modules/node-magickwand',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp: node_modules/node-addon-api/except.gypi not found (cwd: /home/runner/work/node-magickwand/node-magickwand/node_modules/node-magickwand) while reading includes of binding.gyp while trying to load binding.gyp
```

--- a/binding.gyp
+++ b/binding.gyp
@@ -1,7 +1,7 @@
 {
   'target_defaults': {
     'includes': [
-      'node_modules/node-addon-api/except.gypi'
+      'except.gypi'
     ]
   },
   'variables': {
@@ -43,10 +43,6 @@
         '-Wno-deprecated-copy'
       ],
       'msvs_settings': {
-        'VCCLCompilerTool': {
-          # PREfast requires too much memory for Github Actions
-          'EnablePREfast': 'false'
-        },
         'VCLinkerTool': {
           'AdditionalLibraryDirectories': '<(module_root_dir)/deps/ImageMagick-Windows/VisualMagick/lib'
         }

--- a/except.gypi
+++ b/except.gypi
@@ -1,0 +1,25 @@
+{
+  'defines': [ 'NAPI_CPP_EXCEPTIONS' ],
+  'cflags!': [ '-fno-exceptions' ],
+  'cflags_cc!': [ '-fno-exceptions' ],
+  'conditions': [
+    ["OS=='win'", {
+      "defines": [
+        "_HAS_EXCEPTIONS=1"
+      ],
+      "msvs_settings": {
+        "VCCLCompilerTool": {
+          "ExceptionHandling": 1,
+          'EnablePREfast': 'true',
+        },
+      },
+    }],
+    ["OS=='mac'", {
+      'xcode_settings': {
+        'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
+        'CLANG_CXX_LIBRARY': 'libc++',
+        'MACOSX_DEPLOYMENT_TARGET': '10.7',
+      },
+    }],
+  ],
+}

